cmd/crossplane/core/init.go

```diff
@@ -34,6 +33,7 @@ import (
 type initCommand struct {
 	Providers      []string `name:"provider" help:"Pre-install a Provider by giving its image URI. This argument can be repeated."`
 	Configurations []string `name:"configuration" help:"Pre-install a Configuration by giving its image URI. This argument can be repeated."`
+	Namespace      string   `short:"n" help:"Namespace used to set as default scope in default secret store config." default:"crossplane-system" env:"POD_NAMESPACE"`
```

Comment:
Is there any value in naming this something more specific? `--default-secret-namespace` or something? Leaving it just as `namespace` could help if we wanted to default other things to this namespace in future.

Edit: I guess this matches the existing flag used by the package manager, so `--namespace` makes sense.